app/src/main/java/com/example/keybindhelper/MainActivity.kt

```diff
@@ -55,28 +55,28 @@ class MainActivity : AppCompatActivity() {
         setupActionBarWithNavController(navController, appBarConfiguration)
         navView.setupWithNavController(navController)
 
-        DatabaseManager.init(binding.root.context);
+        DatabaseManager.init(binding.root.context)
         CurrentProjectManager.loadFirstProject()
 
 
     }
 
     fun showMenuItems(items:Set<Int>){
         allActionMenuIds.forEach{
-            this.Menu?.findItem(it)!!.isVisible = items.contains(it)
+            this.menu?.findItem(it)!!.isVisible = items.contains(it)
```

Comment:
Variables ideally start with lower case. Not serious just clean coding. 
https://www.w3schools.com/kotlin/kotlin_variables.php

Reply:
This is a good idea, and does conform to best practices.
However, it was already suggested in a previously-submitted code review.  When that occurs, first in wins.